### PR TITLE
Add Rails 6 and Ruby 2.7 to TravisCI targets

### PR DESCRIPTION
**Pull request by mtgto**
Update Travis CI env.

- Add Rails 6.0
- Add Ruby 2.7
- Update Ruby version to latest in minor version
- Drop Ruby 2.4, 2.3 (EOL) https://www.ruby-lang.org/en/downloads/

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,17 +3,23 @@ sudo: false
...
+  - gemfiles/rails_6.0.gemfile
 before_install:
   - "[[ $BUNDLE_GEMFILE =~ rails_4\\.2 ]] && gem uninstall -v '>= 2' -i $(rvm gemdir)@global -ax bundler || true"
   - "[[ $BUNDLE_GEMFILE =~ rails_4\\.2 ]] && gem install bundler -v '< 2' || true"
 before_script:
   - RAILS_ENV=test bundle exec rake db:setup
+matrix:
+  fast_finish: false
+  exclude:
+    - rvm: 2.4.9
```
Comment (mtgto) [inline .travis.yml]: Ruby 6.0 requires Ruby 2.5 or newer
https://guides.rubyonrails.org/upgrading_ruby_on_rails.html#ruby-versions

File: .travis.yml [hunk 2]
```diff
@@ -3,17 +3,23 @@ sudo: false
...
   - gemfiles/rails_5.0.gemfile
   - gemfiles/rails_5.1.gemfile
   - gemfiles/rails_5.2.gemfile
+  - gemfiles/rails_6.0.gemfile
 before_install:
   - "[[ $BUNDLE_GEMFILE =~ rails_4\\.2 ]] && gem uninstall -v '>= 2' -i $(rvm gemdir)@global -ax bundler || true"
   - "[[ $BUNDLE_GEMFILE =~ rails_4\\.2 ]] && gem install bundler -v '< 2' || true"
 before_script:
   - RAILS_ENV=test bundle exec rake db:setup
+matrix:
```
Comment (mtgto) [inline .travis.yml]: https://docs.travis-ci.com/user/build-matrix/

**Comment by l15n** [comment]
Can you rebase against master and try again?

**Comment by mtgto** [comment]
@l15n Update ruby versions to latest (rebase master && force push)

- Q. Ruby 2.4 already ended. Do you want to remove ruby 2.4 from CI?
https://www.ruby-lang.org/en/news/2020/04/05/support-of-ruby-2-4-has-ended/
- `.ruby-version` defines `2.4`. We should update it.

**Comment by l15n** [comment]
> Q. Ruby 2.4 already ended. Do you want to remove ruby 2.4 from CI?

That sounds reasonable 👍

**Comment by mtgto** [comment]
hmm... ci still fails. https://travis-ci.org/github/cookpad/garage/builds/701256431

- ruby 2.7 has bundler v2 itself
- rails 4.2 depends on bundler v1.x

To resolve it, I enforce bundler version for rails 4.2.

**Comment by mtgto** [comment]
Rails 4.2 uses `BigDecimal.new`, which is removed from Ruby 2.7.
I drop Rails 4.2 and Ruby 2.7 from Travis CI target.

**Pull request merged**